presto-docs/src/main/sphinx/sql/create-view.rst

```diff
@@ -42,6 +42,14 @@ Create a view that replaces an existing view::
     SELECT orderkey, orderstatus, totalprice / 4 AS quarter
     FROM orders
 
+Create a view ``test_comment`` over the ``orders`` table, with a comment::
```

Comment:
The view is not called "test_comment" in the example. Please make them consistent. Thanks!